.github/scripts/compile.sh

```diff
@@ -31,11 +31,14 @@ if [ -z "${REMOTE_SOURCES_DIR:-}" ]; then
   SKUPPER_DIR="$WORKING_DIR"
   PROTON_DIR="$WORKING_DIR/proton/app"
   LWS_DIR="$WORKING_DIR/libwebsockets/app"
+  LIBUNWIND_DIR="$WORKING_DIR/libunwind/app"
   # $REMOTE_SOURCES_DIR was not provided, we will have to download the libwebsockets source from ${LWS_SOURCE_URL}
   # Get the libwebsockets source into a libwebsockets.tar.gz file
   # and untar it into the libwebsockets folder
   wget "${LWS_SOURCE_URL}" -O libwebsockets.tar.gz
-  tar -zxf libwebsockets.tar.gz --one-top-level="${LWS_DIR}" --strip-components 1
+  wget "${LIBUNWIND_SOURCE_URL}" -O libunwind.tar.gz
+  tar -zxf libwebsockets.tar.gz --one-top-level="${LWS_DIR}"     --strip-components 1
+  tar -zxf libunwind.tar.gz     --one-top-level="${LIBUNWIND_DIR}" --strip-components 1
```

Comment:
maybe also vertically align the `--strip-components` part, when you are doing the aligning? 🤷🏽